packages/cli/src/commands/init/errors/TemplateAndVersionError.ts

```diff
@@ -0,0 +1,11 @@
+export default class TemplateAndVersionError extends Error {
+  constructor(template: string) {
+    super(
+      `Passing both "version" and "template" is not supported. Templates are in control of "react-native" version. Please choose only one of these, like so:
+      
+      --template ${template}@x.y.z
+      
+      where x.y.z is the version containing the "react-native" you'd like. Check: https://www.npmjs.com/package/${template} for available versions`,
```

Comment:
```suggestion
      where x.y.z is the release of the template that contains the desired "react-native" version. Check the version tab of https://www.npmjs.com/package/${template} for available versions`,
```

Reply:
Sounds way better. Maybe let's also remove double quotes around `react-native` to make it consistent with "The template you select determines the version of react-native used.".